src/components/ShoppingList.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { db } from '../lib/firebase';
+
+const ShoppingList = () => {
+  const [term, setTerm] = useState('');
+
+  const [value, loading, error] = useCollection(
+    db.collection('shoppingLists'),
```

Comment:
The name `shoppingLists` indicates to me that the collection contains lists of items but from the app it appears the collection contains individual items. The naming of the collection could better reflect this.

Reply:
changed it to be singular "shoppingList" to indicate it is just one shopping list that has individual items rather than a list of shopping lists 💯